### PR TITLE
Added Api Functionality

### PR DESCRIPTION
**Pull request by Dmambo**
# FEATURES ADDED

- Implemented the "Refresh" button that will get all scores for a game created by me from the API 
- Implemented the form "Submit" button that will save the score for the game
- Used arrow functions instead of the function keyword
- Used ASYNC and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1 +1,48 @@
...
+  postData(url, data);
+
+  name.value = '';
+  score.value = '';
+});
+
+// window.onload = async () => {
+//   const data = await getData(url);
+//   display(data);
+// };
```
Comment (leonardodiasb) [inline src/index.js]: - [ ] Kindly remove commented-out code. They can make your code looks messy and hard to understand for other developers.

File: src/index.js [hunk 2]
```diff
@@ -1 +1,48 @@
...
+      list.appendChild(li);
+    });
+  }
+};
+
+refresh.addEventListener('click', async () => {
+  const data = await getData(url);
+
+  display(data);
+});
```
Comment (leonardodiasb) [inline src/index.js]: - [ ] The functionality of displaying the data works, but if the user keeps clicking on the 'refresh' button, the data stacks, and then the page displays repeated data. Kindly fix this bug, so when the user clicks on the 'refresh' button, the data is displayed only once, not repeated data.
 - image

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @Dmambo, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Professional PR and README. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

 - [OPTIONAL] It is not a requirement of this part of the project to style your project, but it would be nice to see smaller fonts on your page. Currently, it is not possible to display everything on a normal screen size due to the larger fonts used.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — APPROVED** [review]
Hi @Dmambo, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Linters passed ✔️ 
- Refresh button functionality ✔️ 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**